### ISSUE TITLE
Custom Sizes Attribute is missing in Output

### ISSUE THREAD

**Issue by Der-Lizzard** [ttc-freebies/plugin-responsive-images#568]
Joomla 4.3.3 and PHP 8.1

Tested with plugin versions 4.2.6 and the latest 4.2.7. 

In Frontend, the custom sizes attribute is missing.

If you remove the commented out array_push in line 248 (file: \libraries\Ttc\src\Freebies\Responsive\Helper.php), the custom sizes attribute is shown in frontend.

With commented out
![without](https://github.com/ttc-freebies/plugin-responsive-images/assets/115063400/5c56a207-fc3d-48b2-aea1-899841e18c4b)

Removed commented out
![with](https://github.com/ttc-freebies/plugin-responsive-images/assets/115063400/8ef282ce-d788-4ca6-8e56-15618b34563d)

**Comment by dgrammatiko** [comment]
Pls are welcome, although:
- the sizes would be nice to be user driven (or derived from the breakpoints, the commented code is hardcoded to the default values)
- the sizes are optional (https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images)

**Comment by Der-Lizzard** [comment]
Yes, i know - the sizes are optional - but if there is a field for sizes in the plugin-config, i thought i can use it ;-)

**Comment by dgrammatiko** [comment]
Turns out that I never implemented that feature, the field exists but it's not really used. I will try to publish an update when I get some free time.

Thanks
